src/app-nav-bar/app-nav-bar.js

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+import {useStyletron} from '../styles/index.js';
+import {Cell, Grid} from '../layout-grid/index.js';
+import PrimaryMenuItem from './primary-menu-item.js';
+import MobileNav from './mobile-menu/mobile-nav.js';
+import Logo from './logo.js';
+import UserMenu from './user-menu/user-menu.js';
+import {
+  StyledRoot,
+  StyledSpacing,
+  StyledPrimaryMenuContainer,
+} from './styled-components.js';
+import type {AppNavBarPropsT} from './types.js';
+
+export default function AppNavBar(props: AppNavBarPropsT) {
+  const [css, theme] = useStyletron();
+  const {breakpoints} = theme;
+  const {
+    mainNav = [],
+    userNav = [],
+    appDisplayName,
+    appDisplayNameLink,
+  } = props;
+
+  return (
+    <StyledRoot>
+      {/* Mobile Nav Experience */}
+      <div
+        className={css({
+          [`@media screen and (min-width: ${breakpoints.large}px)`]: {
```

Comment:
Can we use `theme.mediaQuery.large` here?

Reply:
I want to make it customizable in a follow-up change so will leave it as is for now.